### DISCUSSION TITLE
Would possibly like to help :)

### DISCUSSION THREAD

**Discussion by ArieLevental** [triplea-game/triplea#11242] [New Contributor]
Hey!
I'm looking for a cool project to contribute from time to time.
Let me know if you need an extra pair of hands, and how can I help if so :)

**Comment by TheDog-GH** [comment]
Hi @ArieLevental Im not a Dev, but there is some UI stuff that would please most players.
Its been asked for many times, but no one has done/solved it.
Is this the sort of work you are looking for?

https://forums.triplea-game.org/topic/3311/student-group-studying-triplea/2?page=1

**Reply by ArieLevental** [reply]
I'll try and give it a look.
Hopefully I'll be able to do something about it. :)

**Comment by DanVanAtta** [comment]
Yes - we could use help! Do you have any particular areas of interest?

The 2.6 error reports could be a good starting point to get the project up and running locally and to do a first code fix. Some are easier than others. There's a few high-value & high difficulty projects that you could help with, quite a bit for sure!
